<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkToFCompositeFilterWidgetControls</class>
 <widget class="QWidget" name="QmitkToFCompositeFilterWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkToFCompositeFilter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ToF Preprocessing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_ThresholdFilterCheckBox">
     <property name="text">
      <string>Threshold Filter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="m_ThresholdFilterMinValueSpinBox">
     <property name="maximum">
      <number>7000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QxtSpanSlider" name="m_ThresholdFilterRangeSlider">
        <property name="toolTip">
         <string>modify actual seen window by dragging left and right slider. </string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_ThresholdFilterRangeSliderReset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Resets range to histogram minimum and maximum.</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="m_ThresholdFilterMaxValueSpinBox">
     <property name="maximum">
      <number>7000</number>
     </property>
     <property name="value">
      <number>7000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="m_TemporalMedianFilterCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Median Filter (t)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="m_TemporalMedianFilterNumOfFramesSpinBox">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_MedianFilterCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Median Filter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="m_BilateralFilterCheckBox">
     <property name="text">
      <string>Bilateral Filter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Domain σ: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="m_BilateralFilterDomainSigmaSpinBox">
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Range σ:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QSpinBox" name="m_BilateralFilterRangeSigmaSpinBox">
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Kernel Radius:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QSpinBox" name="m_BilateralFilterKernelRadiusSpinBox"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header location="global">qxtspanslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/QmitkToFUtilWidget.qrc"/>
 </resources>
 <connections/>
</ui>
